### ISSUE TITLE
fused batchnorm result in conv

### ISSUE THREAD

**Issue by pilz22** [onnx/keras-onnx#710]
hello，
here I first tried to convert my keras model to onnx format.  when I check the nodes in graph，it seems all conv2d and batchnorm are merged in to one node, the optype is 'conv'. and then i have to convert the onnx model to other model type.  I want to consult whether and how the merged conv keeps the functions of batchnorm?

**Comment by jiafatom** [comment]
We use ConvBatchNormOptimizer to fuse conv+batchnorm, see https://github.com/microsoft/onnxconverter-common/blob/master/onnxconverter_common/optimizer.py#L858
